profiler/src/ProfilerEngine/Datadog.Profiler.Native/IExporter.h

```diff
@@ -13,5 +13,6 @@ class IExporter
 public:
     virtual ~IExporter() = default;
     virtual void Add(Sample const& sample) = 0;
+    virtual void SetEndpoint(std::string runtimeId, uint64_t traceId, std::string endpoint) = 0;
```

Comment:
Suggestion: What about setting IN strings as const `std::string&` ?

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/3015/commits/fec54e15782805f7e9565c90f54f5135668fc49d